rllib/utils/metrics/__init__.py

```diff
@@ -33,6 +33,34 @@
 NUM_MODULE_STEPS_SAMPLED = "num_module_steps_sampled"
 NUM_MODULE_STEPS_SAMPLED_LIFETIME = "num_module_steps_sampled_lifetime"
 
+# Counters for adding and evicting in replay buffers.
+# TODO (Simon): Check, if we should prefix with 'REPLAY_BUFFER'.
+NUM_ENV_STEPS = "num_env_steps"
```

Comment:
Let's rename just a few here:

`NUM_ENV_STEPS` -> `NUM_ENV_STEPS_STORED`
`NUM_AGENT_STEPS` -> `NUM_AGENT_STEPS_STORED`
`NUM_EPISODES` -> `NUM_EPISODES_STORED`

Otherwise, I don't think we need the prefix, b/c we already store the results under a sub-key: "replay_buffer".